scripts/actions/spells/trust/prishe_ii.lua

```diff
@@ -12,7 +12,58 @@ spellObject.onSpellCast = function(caster, target, spell)
 end
 
 spellObject.onMobSpawn = function(mob)
-    xi.trust.message(mob, xi.trust.messageOffset.SPAWN)
+    local psychoAnima  = 3539
+
+    xi.trust.teamworkMessage(mob, {
+        -- Offsets for Lion II and Iroha II messages appear to be incorrect.
```

Comment:
Any more details on this? Whats up?

Reply:
<img width="295" alt="image" src="https://github.com/LandSandBoat/server/assets/1389729/2037bf94-63f6-424b-95db-6e434207462c">

Reply:
```lua
    xi.trust.teamworkMessage(mob, {
        [xi.magic.spell.TENZEN_II] = xi.trust.messageOffset.TEAMWORK_1,
        [xi.magic.spell.NASHMEIRA_II] = xi.trust.messageOffset.TEAMWORK_2,
        [xi.magic.spell.LILISETTE_II] = xi.trust.messageOffset.TEAMWORK_3,
        [xi.magic.spell.ARCIELA_II] = xi.trust.messageOffset.TEAMWORK_4,
        [xi.magic.spell.IROHA_II] = xi.trust.messageOffset.TEAMWORK_5,
    })
```

Reply:
<img width="302" alt="image" src="https://github.com/LandSandBoat/server/assets/1389729/a5620c8f-8e73-4b65-bc21-d476e41a46f1">